buildSrc/build.gradle.kts

```diff
@@ -79,6 +79,10 @@ allprojects {
             name = "kotlin-dev"
             url = uri("https://dl.bintray.com/kotlin/kotlin-dev")
         }
+        maven {
+            name = "Gradle ext releases"
+            url = uri("https://repo.gradle.org/gradle/ext-releases-local")
```

Comment:
`https://repo.gradle.org/gradle/libs-releases` contains `https://repo.gradle.org/gradle/ext-releases-local`, so no need to add a new repository.